server/backend/ios_notifications/ios_apns/iosAPNsRepository.go

```diff
@@ -0,0 +1,143 @@
+package ios_apns
+
+import (
+	"bytes"
+	"encoding/json"
+	"errors"
+	"github.com/TUM-Dev/Campus-Backend/server/backend/ios_notifications/ios_apns/ios_apns_jwt"
+	"github.com/TUM-Dev/Campus-Backend/server/env"
+	"github.com/TUM-Dev/Campus-Backend/server/model"
+	log "github.com/sirupsen/logrus"
+	"golang.org/x/net/http2"
+	"gorm.io/gorm"
+	"io"
+	"net/http"
+	"strconv"
+	"time"
+)
+
+const (
+	BundleId          = "de.tum.tca"
+	ReadIdleTimeout   = 15 * time.Second
+	HTTPClientTimeout = 60 * time.Second
+)
+
+const (
+	APNsDevelopmentURL = "https://api.sandbox.push.apple.com:443"
```

Comment:
Is the `443` part really required since `https://` is 443 by default?

Reply:
Nicht sicher, sie geben die URL allerdings extra so in ihrer documentation an: 
https://developer.apple.com/documentation/usernotifications/setting_up_a_remote_notification_server/sending_notification_requests_to_apns

Reply:
OK, then we keep it that way 🤷🏻‍♂️😂.